### DISCUSSION TITLE
Absolute counts or relative abundance for BC distance matrix

### DISCUSSION THREAD

**Discussion by MonicaSteffi** [vegandevs/vegan#673] [Q&A]
Dear everyone,

I have a dataset of absolute counts and relative abundance obtained from qPCR experiments. I'm interested in calculating the Bray-Curtis dissimilarity to analyze the community structure across different samples.

I plan to use the vegdist function from the vegan package in R.

My main question is:

Can I use relative abundance data directly with vegdist, or do I take the absolute counts abundances to calculate Bray-Curtis dissimilarity?

I'm particularly interested in understanding if there are any pitfalls or considerations I should be aware of when using relative abundance data for this type of analysis. Any insights or recommendations would be greatly appreciated!

Thanks in advance!

**Comment by gavinsimpson** [comment]
What are you interested in modelling? Abundance or compositional difference among levels of the treatment?

Bray Curtis is not a good choice for either of these; the resulting dissimilarity includes a component due to abundance (size) differences between the pair of samples, and a compositional (shape) component. This is problematic if you want to estimate abundance difference or compositional differences as the dissimilarity contains both.

You can use either - it's just a mathematical operation on the data - but you should use the counts (possibly transformed) if you *really* must use it, as this will be as close to the what it was intended for. However, it's going to give you a test that conflates abundance (size) and composition (shape) differences.

Why do you even need to convert your data to dissimilarities?

**Reply by MonicaSteffi** [reply]
> Why do you even need to convert your data to dissimilarities?

I have bacterial counts at different time points for particular experiments. I just want to PCOA plot at the end to know whether microbial communities are different with different time points.

**Reply by gavinsimpson** [reply]
PCoA won't tell you that - you'd need `dbrda()` a Distance-based RDA for that, and you wanting to do a PCoA isn't an answer to why you want dissimilarities. You are throwing away information by converting to dissimilarities and you could avoid this through PCA/CA or RDA/CCA (depending on what you want to look at) just fine. I know that dissimilarities is how everyone seems to want to analyse these molecular data but there is little good reason *in general* to do so, other than it's what everyone does (which isn't a good justification).

**Comment by jarioksa** [comment]
This is not a **vegan** issue: you should consult your scientific community. In general, relative counts are OK and Bray-Curtis may not be optimal if you have very varying totals. (Neither is RDA/PCA in that case.)
